package.json

```diff
@@ -15,6 +15,7 @@
   "pnpm": {
     "overrides": {
       "@types/eslint": "8.4.1",
+      "@embroider/util": "1.11.0",
```

Comment:
This fixed [this build error](https://github.com/cardstack/boxel/actions/runs/5113582654/jobs/9192956817#step:5:159).